exiftool.go

```diff
@@ -122,8 +122,8 @@ func (e *Exiftool) Close() error {
 
 	// Wait for wait to finish or timeout
 	select {
-	case <- ch:
```

Comment:
This code doesn't seem to respect golang coding style. Could you please run gofmt?